pyanaconda/ui/gui/spokes/software_selection.py

```diff
@@ -313,12 +313,12 @@ def status(self):
                     return _("Invalid environment specified in kickstart")
             # we have no packages section in the kickstart and no environment has been set
             elif not self.environment:
-                return _("Nothing selected")
+                return _("Please confirm software selection")
```

Comment:
There is a big difference between `Nothing selected` and `Please confirm software selection`. We shouldn't ask users to confirm something if we are not sure that there is something to confirm. We should have something like this in the `status` property:

```
if not self.completed:
  return _("Please confirm software selection")
```

Reply:
Hm, I am not able to figure how using self.completed can help here.
If we get to the point of "Nothing selected" messages (happens only in case %packages are not specified) then self.completed is either (processing_done is always true at this point)
- True if self.environment exists and is invalid
- False otherwise